PayPalCheckout.podspec

```diff
@@ -12,7 +12,7 @@ Pod::Spec.new do |spec|
   spec.pod_target_xcconfig = { 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64' }
   spec.user_target_xcconfig = { 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64' }
 
-  spec.ios.vendored_frameworks = "PayPalCheckout.xcframework"
+  spec.ios.vendored_frameworks = "PayPalCheckout.xcframework", "CardinalMobile.framework"
```

Comment:
This would only work with a new version, right? since `0.32.0` is already published without the Cardinal framework

Reply:
Yes, this would be merged after a new version was released/we make a final decision on 3rd party Cardinal inclusion. For now it is on hold, should have probably made this PR a draft.